### ISSUE TITLE
On mobile, shelve button should have discernible text

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/murrow.wsu.edu#133]
This is an issue with the WSU Spine, but I'm noting it as an issue for the Murrow launch

`<button id="shelve" type="button"></button>`

**Comment by jeremyfelt** [comment]
Fixed in WSU Spine 1.4.7 https://github.com/washingtonstateuniversity/WSU-spine/pull/411
